pkg/resource/any_test.go

```diff
@@ -30,6 +31,26 @@ func TestNewAnyFromProto(t *testing.T) {
 	r, err := resource.NewAnyFromProto(&protoMd{}, &protoSpec{})
 	assert.NoError(t, err)
 
-	assert.Equal(t, map[string]interface{}{"something": []interface{}{"a", "b", "c"}, "value": "xyz"}, r.Spec())
+	assert.Equal(t, map[string]interface{}{"something": []interface{}{"a", "b", "c"}, "value": "xyz"}, r.Value())
 	assert.Equal(t, "aaa", r.Metadata().ID())
+
+	enc, err := resource.MarshalYAML(r)
+	assert.NoError(t, err)
+
+	out, err := yaml.Marshal(enc)
+	assert.NoError(t, err)
+
+	assert.Equal(t, `metadata:
```

Comment:
```go
    assert.Equal(t, strings.TrimSpace(`
metadata:
  namespace: default
[…]
    something: [a, b, c]
    `) + "\n", string(out))
```

That allows one to keep YAML indented naturally in the test code.

Alternatively, that might be a separate assertion helper if there are more than a few similar comparisons.

Reply:
thanks, updated!